lib/seed/questionnaire_seeder.rb

```diff
@@ -1,16 +1,210 @@
+        type: "group",
+        view_type: "view_group",
+        item: [
+          {
+            type: "display",
+            view_type: "paragraph",
+            text: "monthly_supplies_report.instruction_1"
+          },
+          {
+            type: "group",
+            view_type: "unordered_list_view_group",
+            item: [
+              {
+                type: "display",
+                view_type: "unordered_list_item",
+                icon: "check",
+                icon_color: "#00B849",
+                text: "questionnaire_layout.instruction_1"
+              },
+              {
+                type: "display",
+                view_type: "unordered_list_item",
+                icon: "check",
+                icon_color: "#00B849",
+                text: "questionnaire_layout.instruction_2"
+              }
+            ]
+          },
+          {
+            type: "display",
+            text: "monthly_supplies_report.equipment",
+            view_type: "header"
+          },
+          {
+            type: "display",
+            text: "monthly_supplies_report.available_bp_devices",
+            view_type: "sub_header"
+          },
+          {
+            type: "group",
+            view_type: "input_view_group",
+            item: [
+              {
+                type: "integer",
+                link_id: "monthly_supplies_report.total_available_bp_devices",
+                text: "monthly_supplies_report.stock_on_hand",
+                view_type: "input_field",
+                validations: {
+                  min: 0,
+                  max: 1_000_000
+                }
+              }
+            ]
+          },
+          {
+            type: "display",
+            view_type: "separator"
+          },
+          {
+            type: "display",
+            text: "monthly_supplies_report.drug_stock_losartan",
+            view_type: "sub_header"
+          },
+          {
+            type: "group",
+            view_type: "radio_view_group",
+            link_id: "monthly_supplies_report.losartan_enough_for_next_month",
+            item: [
+              {
+                type: "radio",
+                view_type: "radio_button",
+                text: "questionnaire_layout.yes_lc"
+              },
+              {
+                type: "radio",
+                view_type: "radio_button",
+                text: "questionnaire_layout.no_lc"
+              }
+            ]
+          },
+          {
+            type: "display",
+            view_type: "line_separator"
+          },
+          {
+            type: "display",
+            text: "questionnaire_layout.comments",
+            view_type: "sub_header"
+          },
+          {
+            type: "group",
+            view_type: "input_view_group",
+            item: [
+              {
+                type: "string",
+                link_id: "monthly_supplies_report.comments",
+                text: "questionnaire_layout.empty",
```

Comment:
Ideally we should be able to insert arbitrary strings here, not just keys in the translation files. We are coupling questionnaire to the translations. Can be fixed in the future.